module/importer-ergonode-1/src/Application/DependencyInjection/ErgonodeImporterErgonode1Extension.php

```diff
@@ -32,8 +32,8 @@ public function load(array $configs, ContainerBuilder $container): void
             ->addTag('component.ergonode-importer.product_command_factory_interface');
 
         $container
-            ->registerForAutoconfiguration(AttributeFactoryInterface::class)
-            ->addTag('component.ergonode-importer.attribute_factory_interface');
+            ->registerForAutoconfiguration(ImportAttributeCommandFactoryInterface::class)
+            ->addTag('component.ergonode-importer.attribute_command_factory_interface');
```

Comment:
Since you are changing it anyhow
```suggestion
            ->addTag('component.ergonode-importer.attribute_command_factory');
```